python/lsst/ts/hexrotcomm/enums.py

```diff
@@ -34,6 +34,14 @@ class CommandStatusCode(enum.IntEnum):
     NO_ACK = 2
 
 
+class FrameId(enum.IntEnum):
+    """Frame ID for each message type."""
+
+    COMMAND_STATUS = 1
+    CONFIG = 3
+    TELEMETRY = 2
```

Comment:
I prefer to have the order of:

```
      COMMAND_STATUS = 1
      TELEMETRY = 2
      CONFIG = 3
```

Reply:
OK. I prefer alphabetical order (I wish you had done that in the low-level controller, but I missed it when I reviewed it) but it may be a bit less surprising to sort them numerically.